home/tests/test_display_question_feature.py

```diff
@@ -114,3 +114,15 @@ def test_sorting_answers(self, client, filterType, expected):
             response = client.get(url)
             answer = response.context['answers'].first()
             assert str(answer.content) == expected
+
+        @pytest.mark.django_db
+        def test_invalid_question_url(self, client):
+            url = reverse('question-detail', args=[999])
```

Comment:
maybe a little bit far from now , but maybe instead of 999 (because we will be a huge forum), you can replace it with:
`Question.objects.count()+1`
or 
`Question.objects.last().id+1`

beside that LGTM

Reply:
good idea ido

Reply:
good point, fixed. thanks